src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -405,15 +406,43 @@ def disable_hover(self):
         """Disable hover capabilities in the plotter."""
         self._hover_widget.EnabledOff()
 
+    def __extract_kwargs(self, func_name: Callable, input_kwargs: Dict[str, Any]) -> Dict[str, Any]:
+        """Extracts the keyword arguments from a function signature and returns it as dict.
+
+        Parameters
+        ----------
+        func_name : Callable
+            Function to extract the keyword arguments from. It should be a callable function
+        input_kwargs : Dict[str, Any]
+            Dictionary with the keyword arguments to update the extracted ones.
+
+        Returns
+        -------
+        Dict[str, Any]
+            Dictionary with the keyword arguments extracted from the function signature and
+            updated with the input kwargs.
+        """
+        import inspect
+        signature = inspect.signature(func_name)
+        kwargs = {}
+        for k, v in signature.parameters.items():
+            # We are ignoring positional arguments, and passing everything as kwarg
+            if v.default is not inspect.Parameter.empty:
+                kwargs[k] = v.default
+        for k, v in kwargs.items():
+            # my_args is the arguments passed to the ansys-visualization API
+            if k in input_kwargs:
+                kwargs[k] = input_kwargs[k]
```

Comment:
I know I did suggest the other approach... but maybe this is simpler?

```suggestion
        for k, v in signature.parameters.items():
            # We are ignoring positional arguments, and passing everything as kwarg
            if v.default is not inspect.Parameter.empty:
                kwargs[k] = input_kwargs[k] if k in input_kwargs else v.default
```